test/language/module-code/import-attributes/import-assertion-trlng-comma.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: >
+    AssertClause in ImportDeclaration may contain a trailing comma
+esid: sec-modules
+info: |
+  ImportDeclaration:
+    import ModuleSpecifier[no LineTerminator here] AssertClause;
+
+  AssertClause:
+    with {}
+    with {AssertEntries ,opt}
+
+  AssertEntries:
+    AssertionKey : StringLiteral
+    AssertionKey : StringLiteral , AssertEntries
+
+  AssertionKey:
+    IdentifierName
+    StringLiteral
+features: [import-attributes, globalThis]
+flags: [module]
+---*/
+
+import x from './import-assertion-1_FIXTURE.js' with {test262:'',};
```

Comment:
With the new proposal, unknown import attributes should throw rather than being ignored. Should we require runner to define a no-op `test262` attribute that we can use for testing purposes?

Reply:
Not sure how the runner would add a new attribute -- the set of expected attributes isn't user-extensible is my understanding. 

For this test case, it seems like `type: 'js'` would be sufficient?

Reply:
> Not sure how the runner would add a new attribute -- the set of expected attributes isn't user-extensible is my understanding.

It's defined at the embedder level.

> For this test case, it seems like type: 'js' would be sufficient?

In this specific case it would, however:
- the `type` attribute is not defined in ECMA-262. It's defined by the JSON modules proposal, but only with `type: "json"`
- we need multiple attributes for some tests, for example for testing the order properties in dynamic import are accessed.

Reply:
So hosts will have to implement 'test262-only' attributes?

Reply:
That seems like it'd be prohibitive for runners that aren't a JS engine's shell...